extensions/objects/templates/image_with_regions_editor.html

```diff
@@ -80,3 +78,15 @@ <h4 class="modal-title">
     <button type="button" ng-click="confirmClear()" class="btn btn-danger">Clear Image and Regions</button>
   </div>
 </script>
+<style type='text/css'>
+  input.region-label-editor {
```

Comment:
Use a more specific class that's scoped to this directive, so that there's no danger of it applying outside the scope of this file. Perhaps `image-with-regions-editor input.region-label-editor` might work? (But you'd need to double-check that.)